src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs

```diff
@@ -10,6 +11,23 @@ namespace Raven.Server.Documents.Handlers.Admin
 {
     public class AdminIndexHandler : AdminDatabaseRequestHandler
     {
+        [RavenAction("/databases/*/admin/indexes/replace", "POST")]
```

Comment:
It was requested in the issue http://issues.hibernatingrhinos.com/issue/RavenDB-5622